### PR TITLE
Correct to mclo.gs, Adjust few references.

### PR DESCRIPTION
**Pull request by Kichura**
* Replaces the old paste.gg URL references with mclo.gs instead as the latter allows us to filter out IP addresses for privacy reasons and avoid possible future crashes,

* Corrects the "latest-releases" strings with today's via copies,

* Makes some minor adjustments to eduard's built-in messages.

*(Edits are welcomed if something looks out of place)*

### CODE REVIEW

File: src/main/resources/config.json [hunk 1]
```diff
@@ -1,11 +1,11 @@
...
-    "ViaVersion": "3.0.1",
-    "ViaBackwards": "3.0.1",
-    "ViaRewind": "1.5.0"
+    "ViaVersion": "4.6.1",
+    "ViaBackwards": "4.6.1",
+    "ViaRewind": "2.0.3"
   },
   "tracked-branches": ["master", "dev"],
-  "help-message": "Please explain your issue and patiently wait for a response :)\nConsider also checking pinned messages for common issues (icon in the top right)",
+  "help-message": "Please explain your issue and patiently wait for a response :)\nConsider also checking pinned messages for common issues. *(icon in the top right)*",
```
Comment (kennytv) [inline src/main/resources/config.json:9]: and this

File: src/main/java/eu/kennytv/viaeduard/listener/DumpMessageListener.java [hunk 2]
```diff
@@ -146,7 +146,7 @@ private void sendRequest(final Message message, final String url, final HttpURLC
                 hasProtocolSupport = true;
                 if (isSpigot) {
                     message.addReaction(Emoji.fromUnicode("U+2757")).queue(); // Exclamation mark
-                    EmbedMessageUtil.sendMessage(message.getChannel(), "Via and ProtocolSupport only work together on Paper servers or one of its forks.", Color.RED);
+                    EmbedMessageUtil.sendMessage(message.getChannel(), "Via and ProtocolSupport only work together on Paper servers or one of it's forks!", Color.RED);
```
Comment (kennytv) [inline src/main/java/eu/kennytv/viaeduard/listener/DumpMessageListener.java:149]: revert this line

File: src/main/java/eu/kennytv/viaeduard/listener/DumpMessageListener.java [hunk 3]
```diff
@@ -176,7 +176,7 @@ private void sendRequest(final Message message, final String url, final HttpURLC
 
                     if (hasProtocolSupport && subplatform.equals("ViaBackwards")) {
                         message.addReaction(Emoji.fromUnicode("U+26A1")).queue(); // Lightning
-                        EmbedMessageUtil.sendMessage(message.getChannel(), "Do not use ProtocolSupport and ViaBackwards together, please remove one of them.", Color.RED);
+                        EmbedMessageUtil.sendMessage(message.getChannel(), "Do not use ProtocolSupport and ViaBackwards (+ ViaRewind) together, Please remove one of them.", Color.RED);
```
Comment (kennytv) [inline src/main/java/eu/kennytv/viaeduard/listener/DumpMessageListener.java:179]: lowercase p

Comment (Kichura) [reply src/main/java/eu/kennytv/viaeduard/listener/DumpMessageListener.java]: Changed.

File: src/main/java/eu/kennytv/viaeduard/listener/DumpMessageListener.java (hunk 2 above)
Comment (Kichura) [reply src/main/java/eu/kennytv/viaeduard/listener/DumpMessageListener.java]: Undoed along with the request above.

**Pull request merged**